### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AndreDlm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,24 +44,24 @@ See the [documentation](https://docs.open-systems-pharmacology.org/working-with-
 * More flexible protein localization settings
   * Enzymes/Binding partners: introduction of **fraction expressed** in different compartments
   * Transporters: introduction of **fraction expressed apical/basolateral** (Kidney/Liver/GI) and **fraction expressed blood brain barrier/tissue** (brain)
-* Introduction of **Initial Protein Concentration** as explicit parameter
+* Introduction of **Initial Protein Concentration** as visible parameter
```
Comment (Yuri05) [inline README.md:47]: In the past, initial protein concentration was not a parameter at all.
Initial amount was defined as `M_0 = V_0*relExp*RefConc*f_onto*...`
Now: `M_0 = C_0*V` with `C_0=relExp*RefConc*f_onto*...`
That's what I tried to express with "explicit".

File: README.md [hunk 2]
```diff
@@ -44,24 +44,24 @@ See the [documentation](https://docs.open-systems-pharmacology.org/working-with-
...
+See [Localizations and initial concentrations of enzymes](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#localizations-and-initial-concentrations-of-enzymes) and [Localizations, directions, and initial concentrations of transport proteins](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#localizations-directions-and-initial-concentrations-of-transport-proteins) for detailed descriptions.
 
 * New transporter types:
   * Plasma <=> Blood Cells
   * Plasma <=> Interstitial
   * Bidirectional
 
-* Transport direction (Influx/Efflux/…) can be set in each organ independently
+* Transport direction (Influx/Efflux/Bidirectional) can be set in each organ independently
```
Comment (Yuri05) [inline README.md:56]: Then: `(Influx/Efflux/Bidirectional/Pgp)`

Comment (AndreDlm) [reply README.md:56]: Fine for me

File: README.md (hunk 1 above)
Comment (AndreDlm) [reply README.md:47]: OK I see, 'explicit' might be more appropriate then

**Pull request merged**